lua/wiki/init.lua

```diff
@@ -1,8 +1,12 @@
 local M = {}
 
+local function normalize_path(path)
+  return vim.fn.substitute(vim.fn.expand(vim.g.wiki_root .. "/"), "\\/$", "", nil)
```

Comment:
I believe you meant to pass `vim.g.wiki_root .. "/"` as the argument `path` here?

Reply:
Also, I'm not sure if any of this is really needed. Notice how we already take care of this stuff [here](https://github.com/trev-dev/wiki.vim/blob/b4834fd66993a1286d1590b973f7f830295ec7f0/autoload/wiki/page.vim#L240-L262). I believe we can remove the `filter()` function and instead just use the output from `p:

https://github.com/trev-dev/wiki.vim/blob/b4834fd66993a1286d1590b973f7f830295ec7f0/lua/wiki/init.lua#L22-L23

Reply:
> Also, I'm not sure if any of this is really needed. Notice how we already take care of this stuff [here](https://github.com/trev-dev/wiki.vim/blob/b4834fd66993a1286d1590b973f7f830295ec7f0/autoload/wiki/page.vim#L240-L262). I believe we can remove the filter() function and instead just use the output from `p:

The filter function is used for simply making the full path relative to the wiki_path root. I don't really see why remove it.

Reply:
@saccarosium I think @lervag's built in already abstracts the relative path. Any qualms with just using what's there?

Reply:
I wasn't aware if it built already something. If so probabbly better using those.